benchmark/scripts/test_Benchmark_Driver.py

```diff
@@ -18,7 +18,11 @@
 import time
 import unittest
 
-from StringIO import StringIO
+try:
+    from StringIO import StringIO # py2
+except ModuleNotFoundError:
```

Comment:
I was getting no such module errors from Python 3 because `StringIO` is `io.StringIO` in 3.

Reply:
What I wanted to say is that `ModuleNotFoundError` was introduced in Python 3.6. `ModuleNotFoundError` is a subtype of `ImportError`. It will be more portable to use `ImportError`, so anything from Python 3.0 to Python 3.6 can import `io.StringIO`.